packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx

```diff
@@ -123,7 +162,7 @@ export class ListScrollingExample extends React.Component<IListScrollingExampleP
   private _onRenderCell = (item: IExampleItem, index: number): JSX.Element => {
     return (
       <div data-is-focusable={true}>
-        <div className={css(styles.itemContent, index % 2 === 0 ? styles.itemContentEven : styles.itemContentOdd)}>
+        <div className={index % 2 === 0 ? styles.itemContentEven : styles.itemContentOdd}>
```

Comment:
Can even and odd styles be applied using pure CSS e.g. `nth-child(even)`? It would be more performance since it would offload the calculation to CSS layer and not JS runtime.

Reply:
Got it! I'll refactor it using the selector arg